tests/test_distributed/test_gpu_with_dask/test_gpu_with_dask.py

```diff
@@ -10,6 +10,7 @@
 import pytest
 from hypothesis import given, note, settings, strategies
 from hypothesis._settings import duration
+from packaging.version import parse as parse_version
```

Comment:
This is not in the stdlib, do we need to specify it in our testing environments?

Reply:
Currently, our CI already has `packaging` installed (due to it being a dependency of other packages we use). However, if we were to specify test dependencies in the pyproject.toml, we should definitely list `packaging` as part of the `[test]` suite of optional dependencies.